staging/src/k8s.io/pod-security-admission/api/constants.go

```diff
@@ -48,3 +48,12 @@ const (
 	AuditViolationsAnnotationKey = "audit-violations"
 	EnforcedPolicyAnnotationKey  = "enforce-policy"
 )
+
+var PodSecurityLabels = []string{
```

Comment:
mutable package vars, especially ones used in policy evaluation, aren't ideal... can we avoid them?

Reply:
Switched to a function returning a clone of this.